packages/plugins/src/plugins/metronome/constants.ts

```diff
@@ -0,0 +1,22 @@
+import { NetworkId } from '@sonarwatch/portfolio-core';
+import { Address } from 'viem';
+import BigNumber from 'bignumber.js';
+
+export const PLATFORM_ID = 'metronome';
+export const METRONOME_MARKETS_KEY = 'metronome-markets';
+export const MARKET_ADDRESS: Address =
+  '0x3364f53cb866762aef66deef2a6b1a17c1f17f46';
+
+export const METRONOME_NETWORK_ID = NetworkId.ethereum;
+
+export const PRICE_PER_SHARE_DIVISOR = new BigNumber(10).pow(18);
+export const DEFAULT_DECIMALS = 18;
```

Comment:
```suggestion
export const DEFAULT_DECIMALS = 18;
export const PRICE_PER_SHARE_DIVISOR = new BigNumber(10).pow(DEFAULT_DECIMALS);
```

Shouldn't we eat our own dog-food ?